pkgs/applications/misc/gImageReader/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libuuid
+, sane-backends, podofo, libjpeg, djvulibre, libxmlxx3, libzip, tesseract
+, enchant, intltool, poppler, json-glib
+, ninja
+, python3
+
+# Gtk deps
+# upstream gImagereader supports Qt too
+, gtk3, gobjectIntrospection, wrapGAppsHook
+, gnome3, gtkspell3, gtkspellmm, cairomm
+}:
+
+let
+  variant = "gtk";
+  pythonEnv = python3.withPackages( ps: with ps;[ pygobject3 ] );
+in
+stdenv.mkDerivation rec {
+  name = "gImageReader-${version}";
+  version = "3.2.99";
+
+  src = fetchFromGitHub {
+    owner= "manisandro";
+    repo = "gImageReader";
+    rev = "v${version}";
+    sha256 = "19dbxq83j77lbvi10a8x0xxgw5hbsqyc852c196zzvmwk3km6pnc";
+  };
+
+  nativeBuildInputs = [
+    cmake ninja
+    intltool
+    pkgconfig
+  ]
+  ++ [ wrapGAppsHook gobjectIntrospection ];
+
+  buildInputs = [
+    pythonEnv
+    enchant
+    libxmlxx3
+    libzip
+    libuuid
+    sane-backends
+    podofo
+    libjpeg
+    djvulibre
+    tesseract
+    poppler
+  ]
+  # Gtk specific
+  ++ [
+    gnome3.gtkmm
+    gtkspell3
+    gtkspellmm
+    gnome3.gtksourceview
+    gnome3.gtksourceviewmm
+    cairomm
+    json-glib
+  ];
+
+  # interface type can be where <type> is either gtk, qt5, qt4
+  cmakeFlags = [ "-DINTERFACE_TYPE=${variant}" ];
+
+  meta = with stdenv.lib; {
+    description = "A simple Gtk/Qt front-end to tesseract-ocr";
+    homepage = https://github.com/manisandro/gImageReader;
+    license = licenses.gpl3;
```

Comment:
Looks like it is `gpl3Plus`: https://github.com/manisandro/gImageReader/blob/a8fc69696a9639ee4122ca3fa2ed50514e225afb/gtk/src/main.cc#L8

Reply:
oh nice, didn't know about the difference

Reply:
Yeah, GitHub detects the license based on the `LICENSE` file, which obviously cannot contain the text of future GPL versions. Checking the license headers in the code is usually better.